tests/test_ramp_fitting.py

```diff
@@ -503,6 +638,12 @@ def print_slope_data(slopes):
     print(sdata)
 
 
+def print_slope_dq(slopes):
```

Comment:
Are all of these various print functions used for interactive debugging or what? I don't see them being used in the unit tests.

Reply:
I use these functions to debug and develop tests.  I have added a comment that all functions below the comment are debugging/development functions.